webapp/src/ts/modules/home/home.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AuthService } from '@mm-services/auth.service';
+import { HeaderTabsService } from '@mm-services/header-tabs.service';
+
+@Component({
+  template: '',
+})
+export class HomeComponent implements OnInit {
+
+  constructor(
+    private authService:AuthService,
+    private headerTabsService:HeaderTabsService,
+    private router:Router,
+  ) { }
+
+  ngOnInit() {
+    this.findFirstAuthorizedTab().then(route => {
+      if (!route) {
+        route = 'error';
+      }
+      this.router.navigate([route]);
```

Comment:
Here it needs to send the 403 code for when it's error path, see master: https://github.com/medic/cht-core/blob/449cf15d5fc70948aa9aadc68747326f5b465ee4/webapp/src/js/controllers/home.js#L33

Reply:
Yes sorry, I should add a note about the change. The Angular router does not seems to have an argument to pass any HTTP status code, and it makes sense, we are not doing a real HTTP request, just a redirect within the same page.

Moreover, the `$state.go(state, { code: 403 }, ...` is executed always, not only if there is an error in the path, so it doesn't make sense, but because an actual HTTP is not performed, there is no different in the result in the current master version.

Reply:
Hi!! Yes I understand your explanation on the HTTP, but looks like the webapp is using the same codes for displaying specific error messages see [here](https://github.com/medic/cht-core/blob/32e53c8ae1eefa3a47c38a918b5aa7ee38225009/webapp/src/ts/modules/error/error.component.ts#L24) in this case Error Component takes a route param variable (403) and matches with a specific error message ("Access denied")

For example: When using an user without permissions in tabs and trying to access #/home at gamma, the Home Component redirects to #/error/403
<img width="1572" alt="Screen Shot 2020-12-01 at 10 53 29 am" src="https://user-images.githubusercontent.com/66472237/100695543-fced9500-33c3-11eb-85fd-73366b8cc7e2.png">

> Moreover, the $state.go(state, { code: 403 }, ... is executed always (error or not), 

Yeah, I think this route variable should only be passed during error cases as you point out.

For reference the other scenario: when trying to accessing a path that doesn't exists, it redirects #/error/404
<img width="765" alt="Screen Shot 2020-12-01 at 10 54 10 am" src="https://user-images.githubusercontent.com/66472237/100695608-21497180-33c4-11eb-8297-e6ae47bcad3e.png">

Reply:
**Possibly,** one user that can login but no view on tabs (I know it sounds weird but the system allows it and still good to consider) is some one with only Gateway role 🤔

Reply:
:thinking: you are right about the code in the URL, still doubts about always set the 403 code, I'll do some manual tests...

Reply:
Fixed. I change a little bit how the navigate is executed with or without the error path.